handlers.go

```diff
@@ -50,19 +108,39 @@ func (hh *httpHandlers) ping(w http.ResponseWriter, r *http.Request) {
 
 //goodToGo returns a 503 if the healthcheck fails - suitable for use from varnish to check availability of a node
 func (hh *httpHandlers) goodToGo(writer http.ResponseWriter, req *http.Request) {
-	if _, err := hh.checker(); err != nil {
+	if _, err := hh.checkCanConnectToKafkaProxy(); err != nil {
 		writer.WriteHeader(http.StatusServiceUnavailable)
+		return
 	}
+	if _, err := hh.checkCanConnectToWriters(); err != nil {
+		writer.WriteHeader(http.StatusServiceUnavailable)
+		return
+	}
+}
 
+func checkWriterAvailability(baseURLSlice []string) error {
+	var endpointsToCheck []string
+	for _, baseURL := range baseURLSlice {
+		endpointsToCheck = append(endpointsToCheck, baseURL+"__gtg")
```

Comment:
@tosan88 nice catch:
```GET /__people-rw-neo4j-blue__gtg HTTP/1.1
HTTP/1.1 404 Not Found
```

@scott-ace-newton can you please look into why gtg is returning true in spite of not being able to connect to the writers?